examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p><code>BATCH_SIZE = local_batch_size * strategy.num_replicas_in_sync</code> is the recommended way :)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done! (Here and in all the other places we define BATCH_SIZE)</p><p> </p>